source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -18,16 +18,51 @@ Do _NOT_ let certificates expire. If they expire, the environment becomes non-re
 . Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
 . Once enrollment is complete, click menu:Management[Activate].
 
-.Next steps
-You must also renew Engine certificates.
+[IMPORTANT]
+====
+You must also renew {engine-name} certificates.
+====
```

Comment:
The procedures are relative short. If you combine them, you do not need this callout.

1. Renew the host certificates:
- subprocedure for hosts
2. Renew the Manager certificates:
- subprocedure for Manager